### PR TITLE
Feature - #2865 - Adds (Current project) to target language in select

### PR DESCRIPTION
**Pull request by richmahn**
#### This pull request addresses:

Addresses issue in https://github.com/unfoldingWord-dev/translationCore/issues/2865

#### How to test this pull request:

Go into WA and in the Scripture Pane in the top, remove a language (if already has 3) and then add a language. In the language select drop down you should now see your project's language with `(Current project)` next to it.

---
This change is [Reviewable](https://reviewable.io/reviews/translationcoreapps/scripturepane/72)

### CODE REVIEW

**Comment by PhotoNomad0** [comment]
:lgtm:

---

Review status: 0 of 5 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/translationcoreapps/scripturepane/72#-:-Kyr-uTkLWmmfJbK6UEb:bnfp4nl)*

**Comment by PhotoNomad0** [comment]
Reviewed 2 of 5 files at r1, 3 of 3 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/translationcoreapps/scripturepane/72)*

**Comment by RoyalSix** [comment]
Reviewed 2 of 5 files at r1, 3 of 3 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/translationcoreapps/scripturepane/72)*

**Comment by RoyalSix** [comment]
:lgtm:

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/translationcoreapps/scripturepane/72#-:-Kz0LRvNC25TsD3i89y4:bnfp4nl)*

File: package.json [hunk 1]
```diff
@@ -34,7 +34,13 @@
     "eslint-plugin-react": "^7.4.0",
     "jest": "^21.1.0",
     "react": "^15.6.2",
+    "react-addons-test-utils": "^15.4.2",
```
Comment (RoyalSix) [inline package.json]: Are these changes necessary?

Comment (richmahn) [reply package.json]: Set it up for testing. For TravisCI to run tests on this repo by itself, it is needed.
